pom.xml

```diff
@@ -579,6 +553,92 @@ limitations under the License.]]></inlineHeader>
     </plugins>
   </build>
   <profiles>
+    <profile>
+      <id>jdk11</id>
+      <activation>
+        <jdk>[9..)</jdk>
+      </activation>
+      <build>
+        <pluginManagement>
+          <plugins>
+            <plugin>
+              <artifactId>maven-compiler-plugin</artifactId>
+              <version>3.8.1</version>
+              <configuration>
+                <source>${java.version}</source>
+                <target>${java.version}</target>
+                <release>${java.release.version}</release>
```

Comment:
This property cannot be found.

Reply:
good catch, I had a problem with git and I had not committed the definition, fixed